### DISCUSSION TITLE
Should transfer transaction quantity / amount fields be documented differently? How do transfers and cancellations interact?

### DISCUSSION THREAD

**Discussion by pjohnmeyer** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#192] [Q&A] [answered]
**Examples:**
- [quantity](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/053783239fcf919963b049ba09af8fce31fd4d11/schema/objects/transactions/transfer/StockTransfer.schema.json#L26)
- [amount](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/053783239fcf919963b049ba09af8fce31fd4d11/schema/objects/transactions/transfer/ConvertibleTransfer.schema.json#L26)

All four *Transfer transactions use the phrase "Quantity of non-monetary security units cancelled" or "Amount of monetary value cancelled". My initial suspicion is that these were indavertently copy/pasted from the *Cancel transactions and this should say "transferred".

This got me thinking though, when a transfer occurs, essentially security A is cancelled and "results" in securities B, C...

Should the transaction log contain a transfer and a cancel transaction for security A in this case, or only a transfer?

**Comment by JSv4** [comment, marked as answer]
**TWG Call 5/17/22**

- For purposes of modelling the data and a data transfer format, having the transfer be the terminal transaction makes sense in this case. 
- We need to document this. It would also be helpful to document which transactions are terminal. As part of future developments, it probably makes sense to write a validation tool to do these kinds of referential integrity and internal consistency checks.
